libbeat/outputs/elasticsearch/client.go

```diff
@@ -270,7 +284,13 @@ func bulkEncodePublishRequest(
 			log.Errorf("Failed to encode event meta data: %+v", err)
 			continue
 		}
-		bulkItems = append(bulkItems, meta, event)
+		opType := eventMetaValue(event, "op_type")
```

Comment:
Since we are using the bare string `"op_type"` in a couple of places, lets make it a `const` at the top of this file and add a godoc comment above it.